edx_proctoring/api.py

```diff
@@ -936,6 +936,14 @@ def update_attempt_status(exam_id, user_id, to_status,
                 earned_graded=REJECTED_GRADE_OVERRIDE_EARNED
             )
 
+            certificates_service = get_runtime_service('certificates')
+
+            # invalidate certificate after overriding subsection grade
+            certificates_service.invalidate_certificate(
+                user_id=exam_attempt_obj.user_id,
+                course_key_or_id=exam['course_id']
+            )
+
```

Comment:
I think this would be a good place to add a log with the message similar to what you had in your other PR: "Invalidating certificate for user %d in course %s whose grade dropped below passing threshold due to suspicious proctored exam"

Reply:
done